pkg/iac/terraform/state/test/aws_iam_group/results.golden.json

```diff
@@ -0,0 +1,37 @@
+[
+ {
+  "Id": "acl-0cb1408c0a56522f5",
+  "Type": "aws_network_acl",
```

Comment:
`aws_network_acl` ? I think you do not provided the good statefile for that test